webrender/src/tiling.rs

```diff
@@ -2618,9 +2619,18 @@ impl FrameBuilder {
                                 }
 
                                 // Try to create a mask if we may need to.
-                                if prim_clip_info.is_some() || !clip_info_stack.is_empty() {
+                                if !clip_info_stack.is_empty() {
```

Comment:
We don't need to, as the line above pushes the primitive clip item on to the clip stack, if applicable.